src/web/services/user/profile.rs

```diff
@@ -143,7 +144,44 @@ async fn profile(
     }
 
     // Render the profile template and send to user.
-    return template.in_page(&req, page_title).await;
+    let mut page = template.in_page(&req, page_title.clone()).await?;
+
+    let mut tags = Tags::default();
+    tags.title = page_title.clone();
+    tags.url = format!("/user/{}", id);
```

Comment:
Does this URL work without the rcos.io domain at the beginning of it?

Reply:
I think it does work but I'll go ahead and change it to the same way it is done for meetings (request URI) for consistency.